### DISCUSSION TITLE
Just wanted to know

### DISCUSSION THREAD

**Discussion by BeingCreative09** [tchiotludo/akhq#1438] [General]
Hello sir,
is there any feature available restriction based on cluster showing in the akhq. For example one user:abc is able to see only 3 clusters and other user:sam is able to see all cluster available.

**Comment by tchiotludo** [comment]
not for now, just some design ideas [around](https://github.com/tchiotludo/akhq/discussions/523)

**Reply by BeingCreative09** [reply]
Thank you sir for your quick reponse.
